tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -1572,7 +1572,23 @@ create_compressed_scan_paths(PlannerInfo *root, RelOptInfo *compressed_rel, Comp
 
 	/* create parallel scan path */
 	if (compressed_rel->consider_parallel)
-		ts_create_plain_partial_paths(root, compressed_rel);
+	{
```

Comment:
Since this is the only caller for ts_create_plain_partial_paths we should either move the code into that function or get rid of the function

Reply:
The function is still called in `set_plain_rel_pathlist()`. However, it is used only in non-TSL code. So, I removed the export of the function.